src/test/java/edu/unc/lib/boxc/migration/cdm/services/SipServiceTest.java

```diff
@@ -913,6 +942,99 @@ public void generateSipWithNewCollectionAndRedirectMapping() throws Exception {
         }
     }
 
+    @Test
+    public void generateSipsWithArchivalCollection() throws Exception {
+        testHelper.indexExportData("grouped_gilmer");
+        solrResponseWithPid();
+        testHelper.generateArchivalCollectionDestinationMapping("bdbd99af-36a5-4bab-9785-e3a802d3737e",
+                null, "groupa");
+        testHelper.populateDescriptions("grouped_mods.xml");
+        List<Path> stagingLocs = testHelper.populateSourceFiles("276_185_E.tif", "276_183_E.tif",
+                "276_203_E.tif", "276_241_E.tif", "276_245a_E.tif");
+
+        GroupMappingOptions groupOptions = new GroupMappingOptions();
+        groupOptions.setGroupField("groupa");
+        GroupMappingService groupService = testHelper.getGroupMappingService();
+        groupService.generateMapping(groupOptions);
+        groupService.syncMappings(makeDefaultSyncOptions());
+
+        List<MigrationSip> sips = service.generateSips(makeOptions());
+        assertEquals(3, sips.size());
+
+        MigrationSip sip1 = sips.get(0);
```

Comment:
My understanding is that each SIP (and therefore destination) should have at least one item mapped to it, so we should have that as our goal here. There should be one SIP for the item going to the default destination, one for group1 and one for group2. 

It would be good to add a comment on each of these sip evaluation blocks to indicate which archival collection the sip belongs to, and probably also add an explicit sip.getDestinationPid assertion so we can directly state that each archival group is going to the right destination pid. So it looks like we should be able to check for DEST_UUID1, DEST_UUID2, and "bdbd99af-36a5-4bab-9785-e3a802d3737e".